run.sh

```diff
@@ -83,7 +83,7 @@ api=$base/src/service/http/nebula
 if ! grep -q "$api" <<< `ps aux` ; then
     pushd $api
     npm install
-    NS_ADDR=localhost:9190 NODE_PORT=8088 node node.js
+    NS_ADDR=localhost:9190 NODE_PORT=80 node node.js
```

Comment:
Let's not change this.

Here is the reason:

Many people will have PORT 80 taken on their server, so every new application tends to choose a PORT that is unlikely used already. For you, it's special, because you are using Nebula on a clean machine, and cloud machines usually open 80 to the public only.

Reply:
Ah, ok. Makes sense.